app/views/placements/wizards/add_placement_wizard/_preview_placement_step.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <div class="govuk-width-container">
   <%= render(GovukComponent::NotificationBannerComponent.new(title_text: t(".important"))) do %>
-    <h3 class="govuk-heading-m"><%= t(".this_is_a_preview") %></h3>
+    <h1 class="govuk-heading-m"><%= t(".this_is_a_preview") %></h1>
```

Comment:
Ditto here with regards to our markup for notification banners.

Can we switch to [using `nb.with_heading`](https://govuk-components.netlify.app/components/notification-banner/#notification-banner-with-heading) please?